package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "scripts": {
+    "ci:set-env": "zx ./scripts/ci/set-env.mjs",
```

Comment:
We should probably invoke the script directly on the workflow, since users won't need to use it "manually".

Reply:
Yep, I'll make this change.